README.md

```diff
@@ -1,6 +1,5 @@
 # Hazelcast
 
-[![Gitter](https://img.shields.io/gitter/room/gitterHQ/gitter.svg)](https://gitter.im/hazelcast/hazelcast?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge) 
```

Comment:
I really haven't found a good badge for slack. There's https://slack.hazelcast.com/badge.svg but it  shows zero because of some reason, shields.io decided [not to support slack](https://github.com/badges/shields/issues/536), there are some solutions with scripting (https://github.com/rauchg/slackin) and some other hacky solutions.

Reply:
we use the badge.svg , even though it shows 0. you can just use the slack icon maybe?

Reply:
https://cdn.brandfolder.io/5H442O3W/as/pl546j-7le8zk-ex8w65/Slack_RGB.svg

Reply:
Sure, we can try. Not sure what you meant by:
> we use the badge.svg , even though it shows 0

Reply:
Found a better one, it's on badges.io, just not advertised anywhere. Added, you can see it here - https://github.com/mmedenjak/hazelcast/blob/slack-over-gitter/README.md

Reply:
I mean like this 
![image](https://user-images.githubusercontent.com/510278/90628555-1c4e2100-e21e-11ea-8f55-5be476adc31c.png)

that looks OK too